src/app/js/admin/upload/Upload.js

```diff
@@ -0,0 +1,64 @@
+import React, { PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import RaisedButton from 'material-ui/RaisedButton';
+import ArchiveIcon from 'material-ui/svg-icons/content/archive';
+import Alert from '../../lib/Alert';
+
+import { uploadFile } from './';
+
+export const UploadComponent = ({ onFileLoad, error }) => (
+    <div
+        style={{
```

Comment:
extract those styles outside the component in the same file with a key for each "classes", it'll make the jsx more readable